### DISCUSSION TITLE
ctrl+space (type to place block) doesn't work on chromebook?

### DISCUSSION THREAD

**Discussion by spaciousmind** [ScratchAddons/ScratchAddons#3057] [Help]
Hi, I installed the extension on chrome on my daughters chromebook, ctrl+space doesn't seem to be working at all, I'm not sure if it's chromebook specific or some other issue. anything I can do to troubleshoot? or maybe that hotkey is in use by the OS?

ctrl+f for find works fine, all the other extension features work as far as I can tell.

**Comment by cobaltt7** [comment]
Try Crtl+Shift+I and send screenshots of the window that opens

**Reply by spaciousmind** [reply]
Argh these chromebooks are infuriating. I can't open developer tools because of all the restrictions.

I found the default shortcuts for chromebook and it seems there is a conflict with ctrl+space
I can't see any way to disable this in the OS itself so it seems we are stuck without that feature.
Is there a workaround for this that you know of? perhaps selecting different hotkeys for add block (by typing)

**Reply by CST1229** [reply]
Try using middle-click (if chromebook mouses have that).

**Reply by mxmou** [reply]
This was discussed in #2321. A comment there says that middle clicking is possible by tapping with three fingers.

**Reply by spaciousmind** [reply]
oh yes! three fingers on the mousepad works a treat. Thanks guys!!!

**Reply by sfederici** [reply]
> Try using middle-click (if chromebook mouses have that).

Shift+space doesn't work on my mac. Instead it works shift+left click
